src/studio/src/designer/frontend/dashboard/App.tsx

```diff
@@ -90,10 +99,25 @@ export const App = () => {
             exact={true}
             component={StandaloneDataModelling}
           />
-        </div>
+        </div> : <Grid justifyContent='center'>
+          <AltinnSpinner spinnerText='Venter på svar' />
```

Comment:
I think this pattern we're introducing here is a bit odd - but I also see the need to fix this immediately. This is definitely better than the situation today. 

Not great to get into this situation, without a good way of getting out of it. Should we perhaps also show a message to the user when this situation occurs (user object not being populated for some unknown reason), and suggest they log out and log back in again?

Reply:
This is the intent with the log out button appearing after 5 seconds... Right now this spinner is hardly visible unless the page is taking an unusual amount of time to load.
Then the Log out button will appear after 5 secs to give the user an option to log out and try again.

Reply:
Right now, the logout button will just appear with no explanation. I'm just not sure its obivous to users that logging out and in again will fix the issue - so some more explanation would be helpful I think. But probably not important, I guess this is not a common scenario?